File: Java/SpaceVIL/sandbox/src/main/java/sandbox/View/Simple.java
package sandbox.View;

import java.awt.Color;
import com.spvessel.spacevil.ActiveWindow;
import com.spvessel.spacevil.BlankItem;
import com.spvessel.spacevil.Ellipse;
import com.spvessel.spacevil.Decorations.Shadow;
import com.spvessel.spacevil.Flags.SizePolicy;
import com.spvessel.spacevil.Core.Position;
import com.spvessel.spacevil.Core.Size;
import com.spvessel.spacevil.Flags.MSAA;
import com.spvessel.spacevil.Flags.RenderType;
import com.spvessel.spacevil.WindowManager;

public class Simple extends ActiveWindow {
    @Override
    public void initWindow() {
        WindowManager.setRenderType(RenderType.IfNeeded);
        setSize(400, 400);
        setAntiAliasingQuality(MSAA.MSAA8x);

        Ellipse ellipse = new Ellipse(64);
        ellipse.setSizePolicy(SizePolicy.Expand, SizePolicy.Expand);
        ellipse.setBackground(255, 155, 155);
        ellipse.setMargin(100, 100, 100, 100);
        ellipse.effects().add(new Shadow(10, new Size(140, 140), new Color(255, 0, 255, 255)));
        ellipse.effects().add(new Shadow(10, new Size(120, 120), new Color(0, 0, 255, 255)));
        ellipse.effects().add(new Shadow(10, new Size(100, 100), new Color(128, 255, 255, 255)));
        ellipse.effects().add(new Shadow(10, new Size(80, 80), new Color(0, 255, 0, 255)));
        ellipse.effects().add(new Shadow(10, new Size(60, 60), new Color(255, 255, 128, 255)));
        ellipse.effects().add(new Shadow(10, new Size(40, 40), new Color(255, 128, 0, 255)));
        ellipse.effects().add(new Shadow(10, new Size(20, 20), new Color(255, 0, 0, 255)));

        addItem(ellipse);
    }
}

File: CSharp/SpaceVIL/sandbox/View/Simple.cs
using System.Drawing;
using SpaceVIL;
using SpaceVIL.Core;
using SpaceVIL.Decorations;

namespace View
{
    public class Simple : ActiveWindow
    {

        public override void InitWindow()
        {
            WindowManager.SetRenderType(RenderType.IfNeeded);
            SetSize(400, 400);
            SetAntiAliasingQuality(MSAA.MSAA8x);

            Ellipse ellipse = new Ellipse(64);
            ellipse.SetSizePolicy(SizePolicy.Expand, SizePolicy.Expand);
            ellipse.SetBackground(255, 155, 155);
            ellipse.SetMargin(100, 100, 100, 100);
            ellipse.Effects().Add(new Shadow(10, new SpaceVIL.Core.Size(140, 140), Color.FromArgb(255, 0, 255)));
            ellipse.Effects().Add(new Shadow(10, new SpaceVIL.Core.Size(120, 120), Color.FromArgb(0, 0, 255)));
            ellipse.Effects().Add(new Shadow(10, new SpaceVIL.Core.Size(100, 100), Color.FromArgb(128, 255, 255)));
            ellipse.Effects().Add(new Shadow(10, new SpaceVIL.Core.Size(80, 80), Color.FromArgb(0, 255, 0)));
            ellipse.Effects().Add(new Shadow(10, new SpaceVIL.Core.Size(60, 60), Color.FromArgb(255, 255, 128)));
            ellipse.Effects().Add(new Shadow(10, new SpaceVIL.Core.Size(40, 40), Color.FromArgb(255, 128, 0)));
            ellipse.Effects().Add(new Shadow(10, new SpaceVIL.Core.Size(20, 20), Color.FromArgb(255, 0, 0)));

            AddItem(ellipse);
        }
    }
}